muxpi/dut.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+echo "Setting up DUT" 
+#Clear muxpi display and print 'DUT'
+stm_armv7 -clr
+stm_armv7 -print "DUT"
+
+#Make the SD card adapter emulate a real SD-card. Note that the SD card wont be visible on muxpi when its on DUT mode.
+stm_armv7 -dut
+echo "Waiting for the screens... This takes a short while"
```

Comment:
It would be enough with just the first part:
```suggestion
echo "Waiting for the screens..."
```